default/stringtables/en.txt

```diff
@@ -2610,6 +2613,29 @@ Create Binary Save Files
 OPTIONS_USE_XML_ZLIB_SERIALIZATION
 Use Compression for XML Save Files 
 
+OPTIONS_CREATE_PERSISTENT_CONFIG
+Retain modified settings across versions
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_TOOLTIP_TITLE
+Create Persistent Config (Advanced Feature)
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_TOOLTIP_DESC
+'''Uses your current config settings as persistent default settings.
+Such settings take priority over any settings in config.xml, but not the command line.
+Only stores settings that have been modified from stock values.
+Any existing persistent config file is overwritten.
+
+Settings may not always be valid across versions.
+
+See http://freeorion.org/index.php/Config.xml for detailed info on persistent_config.xml'''
```

Comment:
Should probably put relevant info in the pedia, rather than requiring user to look up a website.

Shouldn't be difficult to add the path to the config files to a generated pedia article.